client/src/components/General/Signup/Component.jsx

```diff
@@ -20,7 +21,8 @@ import {
   emailRequired,
   passwordFormat,
   passwordLength,
-  passwordRequired
+  passwordRequired,
+  passwordConfirmed
```

Comment:
you import `passwordConfirmed` but there is no inside `client/src/utils/validationRules.js`

Reply:
fixed.